packages/editor/src/index.ts

```diff
@@ -16,14 +16,18 @@ import type { PartialDeep, InquirerReadline } from '@inquirer/type';
 type EditorConfig = {
   message: string;
   default?: string;
-  postfix?: string;
   waitForUseInput?: boolean;
   validate?: (value: string) => boolean | string | Promise<string | boolean>;
   theme?: PartialDeep<Theme>;
+  file?: IFileOptions;
 };
 
 export default createPrompt<string, EditorConfig>((config, done) => {
-  const { waitForUseInput = true, postfix = '.txt', validate = () => true } = config;
+  const {
+    waitForUseInput = true,
+    file: { postfix = '.txt', ...fileProps } = {},
```

Comment:
To keep it backward compatible without too much hassle:

```suggestion
    file: { postfix = config.postfix ?? '.txt', ...fileProps } = {},
```

Reply:
done 👍